core/string/ustring.cpp

```diff
@@ -34,12 +34,12 @@
 #include "core/math/color.h"
 #include "core/math/math_funcs.h"
 #include "core/os/memory.h"
+#include "core/string/node_path.h"
```

Comment:
You include `node_path.h` but this is actually defined in `string_names.h`.

Reply:
Oh nice catch, I moved this to string name in the end.